docker-compose.component.override.yml

```diff
@@ -16,6 +18,7 @@ services:
       - FAKE_SPINE_PRIVATE_KEY
       - FAKE_SPINE_CA_STORE
       - INBOUND_SERVER_BASE_URL=https://inbound
+      - MHS_LOG_LEVEL=DEBUG
```

Comment:
should we use DEBUG level here? Isn't like that that some sensitive data would be visible in the logs? I remember that this should not be below INFO level

Reply:
Low risk since its only for testing, but changed to INFO for best practice 👍